AUTHORS

```diff
@@ -1,6 +1,7 @@
 The following people, in alphabetical order, have either authored or signed
 off on commits in the Cilium repository:
 
+                                        
```

Comment:
The script generated an empty line?

Reply:
Yeah it's started doing that recently, I first noticed during one of the last two release runs. I have a sneaking suspicion there may be some commit with an empty author or something else which the script doesn't like, but I haven't gone through the trouble to identify the commit yet.